substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSections.java

```diff
@@ -373,8 +359,11 @@ public void installDebugInfo(DebugInfoProvider debugInfoProvider) {
                  */
                 Range subRange = new Range(fileNameAtLine, filePathAtLine, classNameAtLine, methodNameAtLine, "", "", stringTable, loAtLine, hiAtLine, line, primaryRange);
                 addSubRange(primaryRange, subRange);
+                try (DebugContext.Scope s = debugContext.scope("Subranges")) {
```

Comment:
Here we are logging with a scope nested within the scope you get from `debugCodeInfo.debugContext((debugContext) ->`. The whole scope inside here is:
```
`ForkJoinPool-?-worker-?.CreateBootImage.NativeBootImage.build.DebugCodeInfo.Subranges`
```